packages/orbit-components/src/Checkbox/Checkbox.ct.tsx

```diff
@@ -1,12 +1,161 @@
 import * as React from "react";
 import { test, expect } from "@playwright/experimental-ct-react";
+import { defaultTokens } from "@kiwicom/orbit-design-tokens";
 
-import CheckboxStory from "./Checkbox.ct-story";
+import { CheckboxStory, CheckboxWithTooltipStory } from "./Checkbox.ct-story";
 
 test.describe("visual Checkbox", () => {
+  const { breakpointLargeMobile } = defaultTokens;
+
   test("Checkbox", async ({ mount }) => {
     const component = await mount(<CheckboxStory />);
+    await expect(component).toHaveScreenshot();
+  });
+
+  test("Checkbox - disabled, active tooltip on desktop", async ({ mount, viewport }) => {
+    /**
+     * This command skips the test if the viewport width is smaller than largeMobile.
+     * Condition inside is based on the viewport width,
+     * which is specified for each of the device in playwright-ct.config.ts file.
+     * Using that we can distinguish mobileSmall, mobileMedium devices and larger devices.
+     */
+    test.skip(
+      viewport !== null && viewport.width < breakpointLargeMobile,
+      "This feature is largeMobile, tablet and desktop-only",
+    );
```

Comment:
What do you think about keeping something like this at the beggining of the tests and reuse it across?

```ts

const skipIfViewportSmallerThan = (viewport, width, message) => {
  test.skip(
    viewport !== null && viewport.width < width,
    message,
  );
};

const skipIfViewportLargerThanOrEqual = (viewport, width, message) => {
  test.skip(
    viewport !== null && viewport.width >= width,
    message,
  );
};
  ```